modules/core/src/lib/deck.ts

```diff
@@ -672,11 +672,11 @@ export default class Deck<ViewsT extends ViewOrViews = null> {
     this.effectManager!.addDefaultEffect(effect);
   }
 
-  _addDefaultShaderModule(module: ShaderModule) {
+  _addDefaultShaderModule(module: ShaderModule<any>) {
```

Comment:
Can `ShaderModule<unknown>` be used?

To me, a use of `any` sends a signal the the developer gave up on types at this point, `unknown` signals intent. That said there are cases where `any` is necessary.

Reply:
Updated to `ShaderModule<Record<string, unknown>>`